sdk/src/feature_set.rs

```diff
@@ -554,6 +554,10 @@ pub mod enable_program_redeployment_cooldown {
     solana_sdk::declare_id!("J4HFT8usBxpcF63y46t1upYobJgChmKyZPm5uTBRg25Z");
 }
 
+pub mod commission_updates_only_allowed_in_first_half_of_epoch {
+    solana_sdk::declare_id!("ComffCcTMthDAx5AbFA4LeRzGRUB4Z1wNhXz354zvAmT");
```

Comment:
`time solana-keygen grind --starts-with rug:1` exits in ~2 seconds on my machine. :)

Reply:
well, that is certainly more fun. I'll spend an hour on `norug:1` and see where I get

Reply:
`noRuG2kzACwgaY7TVmLRnUNPLKNVQE1fb7X55YWBehp` :sunglasses: